src/angular/src/app/spark-core-angular/directives/inputs/sprk-button/sprk-button.directive.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { SprkButtonDirective } from './sprk-button.directive';
+
+@Component({
+  selector: 'sprk-test',
+  template: `
+  <button
+    additionalClasses="sprk-u-man"
+    sprkButton></button>`
+})
+class TestComponent {}
+
+describe('Spark Button Directive', () => {
+  let component: TestComponent;
+  let fixture: ComponentFixture<TestComponent>;
+  let buttonElement: HTMLElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [SprkButtonDirective, TestComponent]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(TestComponent);
+    component = fixture.componentInstance;
+    buttonElement = fixture.nativeElement.querySelector('button');
+    fixture.detectChanges();
+  }));
+
+  it('should create itself', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should add classes if additionalClasses has a value', () => {
+    fixture.detectChanges();
+    expect(buttonElement.classList.toString()).toEqual(
+      'sprk-c-Button sprk-u-man'
```

Comment:
Where is the setting of additionalClasses to "sprk-u-man"?

Reply:
its up in the declaration